project/plugins.sbt

```diff
@@ -1,11 +1,11 @@
 /*
  * Copyright (C) 2015-2017 Lightbend Inc. <https://www.lightbend.com>
  */
-addSbtPlugin("com.typesafe.play" % "interplay" % sys.props.get("interplay.version").getOrElse("1.1.2"))
+addSbtPlugin("com.typesafe.play" % "interplay" % sys.props.get("interplay.version").getOrElse("2.0.1"))
```

Comment:
Use interplay 1.3.15 since 2.0.x adds Scala 2.13 which is not supported by Play 2.6.